### PR TITLE
https://issues.redhat.com/browse/ACM-7110 Updating the Important note

### PR DESCRIPTION
**Pull request by dockerymick**
https://issues.redhat.com/browse/ACM-7110

### CODE REVIEW

File: clusters/cluster_lifecycle/cluster_proxy_addon.adoc [hunk 1]
```diff
@@ -3,7 +3,12 @@
 
 In some environments, a managed cluster is behind a firewall and cannot be accessed directly by the hub cluster. To gain access, you can set up a proxy add-on to access the `kube-apiserver` of the managed cluster to provide a more secure connection.
 
-*Important:* The `cluster-proxy-addon` does not work when a hub cluster or managed cluster has a link:https://docs.openshift.com/container-platform/4.12/networking/enable-cluster-wide-proxy.html[cluster-wide-proxy] configuration.
+*Important:* The `cluster-proxy-addon` does not work when another proxy configuration is on your hub cluster or managed clusters. The following scenarios cause include:
```
Comment (dockerymick) [inline clusters/cluster_lifecycle/cluster_proxy_addon.adoc]:
```suggestion
*Important:* The `cluster-proxy-addon` does not work when another proxy configuration is on your hub cluster or managed clusters. The following scenarios causes this:
```

File: clusters/cluster_lifecycle/cluster_proxy_addon.adoc [hunk 2]
```diff
@@ -3,7 +3,7 @@
 
 In some environments, a managed cluster is behind a firewall and cannot be accessed directly by the hub cluster. To gain access, you can set up a proxy add-on to access the `kube-apiserver` of the managed cluster to provide a more secure connection.
 
-*Important:* The `cluster-proxy-addon` does not work when a hub cluster or managed cluster has a link:https://docs.openshift.com/container-platform/4.12/networking/enable-cluster-wide-proxy.html[cluster-wide-proxy] configuration.
+*Important:* There must be one cluster-wide proxy configuration on your hub cluster.
```
Comment (xuezhaojun) [inline clusters/cluster_lifecycle/cluster_proxy_addon.adoc]: should here change to `must not` ?

Comment (xuezhaojun) [reply clusters/cluster_lifecycle/cluster_proxy_addon.adoc]: Or we can remove this line? Since the limitation is recorded on the `known_issues` page.

Comment (dockerymick) [reply clusters/cluster_lifecycle/cluster_proxy_addon.adoc]: @xuezhaojun I'm confused here. The doc is about using the proxy add-on on and then users are instructed to configure the proxy for both a hub cluster and a managed cluster. Should there only be one proxy on the managed cluster? Why do we want to remove the note completely? Here is a rewrite with your suggestion:

"*Important:* There must not be a cluster-wide proxy configuration on your hub cluster."

Comment (xuezhaojun) [reply clusters/cluster_lifecycle/cluster_proxy_addon.adoc]: The cluster-proxy addon aims to set up a tunnel from  an agent to a hub, (now it looks like the name should be `cluster-tunnel`, which may make more sense.)

The tunnel's ends are inside the hub and agent.

---

However, the cluster-wide proxy aims to support when there is a proxy between the hub and agents.

Normally the proxy-server here is in front of the hub or an agent.

---

If there is a proxy, the cluster-proxy addon can't set up the tunnel.

Comment (xuezhaojun) [reply clusters/cluster_lifecycle/cluster_proxy_addon.adoc]: The updated version looks good to me!

Comment (dockerymick) [reply clusters/cluster_lifecycle/cluster_proxy_addon.adoc]: @xuezhaojun when you say "agent", are you referring to the managed cluster?

Comment (dockerymick) [inline clusters/cluster_lifecycle/cluster_proxy_addon.adoc]:
```suggestion
*Important:* There must not be a cluster-wide proxy configuration on your hub cluster.
```

Comment (dockerymick) [reply clusters/cluster_lifecycle/cluster_proxy_addon.adoc]: Nvm

**Review by oafischer — APPROVED** [review]
Looks great, thanks!

File: clusters/cluster_lifecycle/cluster_proxy_addon.adoc (hunk 2 above)
Comment (dockerymick) [reply clusters/cluster_lifecycle/cluster_proxy_addon.adoc]: @xuezhaojun it also seems like we might want to make some clarifications based on your [comment](https://github.com/stolostron/rhacm-docs/pull/5703#discussion_r1377142685). I recommend that you create a new doc issue to doc the clarifications for 2.9 doc refresh

Comment (xuezhaojun) [reply clusters/cluster_lifecycle/cluster_proxy_addon.adoc]: Thanks for the update! I will submit a new doc issue later to clarify.

**Pull request merged**